README.md

```diff
@@ -7,9 +7,9 @@ It contains two kinds of plugins to the [clang compiler](http://clang.llvm.org/)
 
 - analyzer plugins use the internal APIs of [clang analyzer](http://clang-analyzer.llvm.org/) to find bugs and report them;
 
-- frontend plugins process the syntax of source files directly to accomplish more general tasks; specifically, we have developed a clang-to-ocaml bridge to make code analyses easier.
+- frontend plugins process the syntax of source files directly to accomplish more general tasks; specifically, we have developed a clang-to-ocaml bridge to make code analyzes easier.
```

Comment:
Afaik analyses is the plural of analysis: http://grammarist.com/spelling/analyses-vs-analyzes/